### PR TITLE
[10.0][MIG] port purchase_product_multi_add to 10.0

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
Port PR https://github.com/OCA/purchase-workflow/pull/238 on V8.0 to V10.0

### CODE REVIEW

File: purchase_product_multi_add/wizard/purchase_import_products.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        @rtype: dict
+        """
+        return {
+            'order_id': purchase.id,
+            'name': item.product_id.name,
+            'product_id': item.product_id.id,
+            'product_qty': item.quantity or 1,
+            'product_uom': item.product_id.uom_po_id.id,
+            'price_unit': 0.0,
+            'date_planned': datetime.today().strftime(
```
Comment (lmignon) [inline purchase_product_multi_add/wizard/purchase_import_products.py]: @Cedric-Pigeon 
```python
'data_panned': fields.Datetime.now()
```

File: purchase_product_multi_add/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+=====
+
+This module add a button on purchase order to import several products filtered
+on the supplier you have selected in your purchase order.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{purchase-workflow}/{8.0}
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (lmignon) [inline purchase_product_multi_add/README.rst]: These commented lines should be removed

File: purchase_product_multi_add/README.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Usage
+=====
+
+This module add a button on purchase order to import several products filtered
+on the supplier you have selected in your purchase order.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{purchase-workflow}/{8.0}
```
Comment (lmignon) [inline purchase_product_multi_add/README.rst]: @Cedric-Pigeon type... 10.0

File: purchase_product_multi_add/wizard/purchase_import_products.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        @rparam: values to create purchase order line with the product item and
+        the right quantity
+        @rtype: dict
+        """
+        return {
+            'order_id': purchase.id,
+            'name': item.product_id.name,
+            'product_id': item.product_id.id,
+            'product_qty': item.quantity or 1,
+            'product_uom': item.product_id.uom_po_id.id,
```
Comment (lmignon) [inline purchase_product_multi_add/wizard/purchase_import_products.py:61]: @Cedric-Pigeon A nice to have should be to have able to select the product_uom on `purchase.import.products.items`

**Review by lmignon — APPROVED** [review]
A little comment.... 
Nevertheless :+1:  (Code review + tested on runbot)

**Comment by rousseldenis** [comment]
@Cedric-Pigeon Can you check @lmignon comments to move forward on this?

**Comment by Cedric-Pigeon** [comment]
@rousseldenis done.

File: purchase_product_multi_add/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# © 2016 Denis Roussel, ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Purchase Product Multi Add",
+    'summary': """
+        This module allows a shortcut to add purchase.order.line by selecting
+        product into a wizard for the given suplier
+    """,
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
```
Comment (nikul-serpentcs) [inline purchase_product_multi_add/__manifest__.py]: https://github.com/OCA/purchase-workflow

**Review by nikul-serpentcs — COMMENTED** [review]
Nitpicking

File: purchase_product_multi_add/__manifest__.py (hunk 5 above)
Comment (Cedric-Pigeon) [reply purchase_product_multi_add/__manifest__.py]: @nikul-serpentcs Done. Thanks

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1: 

Add README like this https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

**Comment by Cedric-Pigeon** [comment]
@nikul-serpentcs Just upgraded the readme structure

**Comment by rousseldenis** [comment]
@Cedric-Pigeon Could you see flake8 errors ? https://travis-ci.org/OCA/purchase-workflow/jobs/591457702

**Comment by Cedric-Pigeon** [comment]
@rousseldenis @nikul-serpentcs This one is :green_apple:

**Comment by Cedric-Pigeon** [comment]
@OCA/procurement-purchase-maintainers Does any one can trigger the merge please?

**Pull request merged**